connectivity/check/action.go

```diff
@@ -119,6 +124,37 @@ func (a *Action) Destination() TestPeer {
 	return a.dst
 }
 
+func (a *Action) startMonitors(opts ...string) {
+	// Launch monitor on each of the Cilium pods.
+	for _, pod := range a.test.ctx.ciliumPods {
+		a.monitors = append(a.monitors, a.NewMonitor(pod, opts...))
+	}
+
+	// Wait that each monitor has started execution.
+	// Waiting in the launch order after all the monitors have
+	// been launched minimizes the overall wait time.
+	for _, monitor := range a.monitors {
+		monitor.Wait()
+	}
+}
+
+func (a *Action) stopMonitors() {
+	for _, monitor := range a.monitors {
+		monitor.Stop()
+	}
+}
+
+func (a *Action) printMonitors() {
+	for _, monitor := range a.monitors {
+		output, err := monitor.Output()
+		if err != nil {
+			a.Warnf("Monitor on pod %s failed: %w", monitor.Name(), err)
+		}
+		a.Logf("\n------------------------------------------------------------------------------\nMonitor output for pod %s:\n%s",
```

Comment:
It looks like this with the typical Cilium monitor output:
```
  
  
------------------------------------------------------------------------------
Monitor output for pod kube-system/cilium-8d9p2:
------------------------------------------------------------------------------
CPU 00: MARK 0x4fb5934 FROM 3930 from-stack: 74 bytes (74 captured), state new, identity 1->0, orig-ip 0.0.0.0
level=info msg="Initializing dissection cache..." subsys=monitor
Ethernet	***Contents=[..14..] Payload=[..62..] SrcMAC=7e:88:d4:28:d1:80 DstMAC=ee:39:0e:84:db:9e EthernetType=IPv4 Length=0***
IPv4	***Contents=[..20..] Payload=[..40..] Version=4 IHL=5 TOS=0 Length=60 Id=26262 Flags=DF FragOffset=0 TTL=64 Protocol=TCP Checksum=48636 SrcIP=10.240.0.35 DstIP=10.240.0.39 Options=[] Padding=[]***
TCP	***Contents=[..40..] Payload=[] SrcPort=50034 DstPort=8080(http-alt) Seq=2350296708 Ack=0 DataOffset=10 FIN=false SYN=true RST=false PSH=false ACK=false URG=false ECE=false CWR=false NS=false Window=64240 Checksum=5720 Urgent=0 Options=[..5..] Padding=[]***
------------------------------------------------------------------------------
CPU 00: MARK 0x4fb5934 FROM 3930 to-endpoint: 74 bytes (74 captured), state new, identity 1->3775, orig-ip 10.240.0.35, to endpoint 3930
Ethernet	***Contents=[..14..] Payload=[..62..] SrcMAC=7e:88:d4:28:d1:80 DstMAC=ee:39:0e:84:db:9e EthernetType=IPv4 Length=0***
IPv4	***Contents=[..20..] Payload=[..40..] Version=4 IHL=5 TOS=0 Length=60 Id=26262 Flags=DF FragOffset=0 TTL=64 Protocol=TCP Checksum=48636 SrcIP=10.240.0.35 DstIP=10.240.0.39 Options=[] Padding=[]***
```

Reply:
I see the first newline is too much, but otherwise I kind of like how this looks.

Reply:
You mean like this:
```
		a.Log("------------------------------------------------------------------------------")
		a.Logf("Monitor output for pod %s:", monitor.Name())
		a.Logf("%s", output)
```
??

Reply:
If the `---` would be rendered according to the amount of cols of the terminal, that could make sense. Otherwise, it just looks like visual clutter to me. Why not just a newline to separate?